src/RemoteTech/Modules/ModuleRTAntenna.cs

```diff
@@ -430,17 +430,21 @@ private bool GetShieldedState()
         {
             PartModule FARPartModule = GetFARModule();
 
-            if (FARPartModule != null) {
-                try {
-                    FieldInfo fi = FARPartModule.GetType ().GetField ("isShielded");
-                    return (bool)(fi.GetValue (FARPartModule));
-                } catch (Exception e) {
-                    RTLog.Notify ("GetShieldedState: {0}", e);
-                    return false;
+            if (FARPartModule != null)
+            {
+                try
+                {
+                    FieldInfo fi = FARPartModule.GetType().GetField("isShielded");
+                    return (bool)(fi.GetValue(FARPartModule));
+                }
+                catch (Exception)
+                {
+                    // otherwise go further and try to get the stock shielded value
```

Comment:
I think we should keep the notify in here. exceptions are expensive and i think we should know when we are catching a ton of them.